crates/bevy_transform/src/components/children.rs

```diff
@@ -7,6 +7,7 @@ use bevy_reflect::Reflect;
 use smallvec::SmallVec;
 use std::ops::Deref;
 
+/// Contains references to the child entities of this entity
```

Comment:
To me, references makes me think of `&T`.

Possibly references is the right term; if it is, we should ensure we're consistent about it (but that's not this PR's problem)